scripts/first-run.py

```diff
@@ -90,6 +93,12 @@ def lose(msg, e=None):
 with open("README.md", "w") as fh:
     fh.write(readme)
 
+# Pin the dependencies
+print("Pinning dependencies")
+dependencies = subprocess.check_output(["pip", "freeze"])
+with open("requirements.txt", "wb") as dependency_file:
+    dependency_file.write(dependencies)
```

Comment:
There must be a more elegant way to do this that doesn't rely on subprocess.

Reply:
Sure. I mean run shell code in the Python script. `os.system("pip freeze > requirements.txt")`

Reply:
Anyway. I'm golfing. This is fine and it works.